docs/monitoring.rst

```diff
@@ -112,3 +114,50 @@ endpoint name ``brokers.list`` with the metric name ``request-rate`` to get
                                  ``Content-Type: application/vnd.kafka.binary.v1+json`` header
   ``topics.list``                ``GET /topics``
   ============================== ===================================================================
+
+  For API v2:
+  
+  ==================================  ======================================================================
+  ``consumer.commit-offsets+v2``      ``POST /consumers/{group}/instances/{instance}/offsets``
+                                      with ``Content-Type: application/vnd.kafka.v2+json`` header
+  ``consumer.committed-offsets+v2``   ``GET /consumers/{group}/instances/{instance}/offsets``
+                                      with ``Accept: application/vnd.kafka.v2+json`` header
+  ``consumer.create+v2``              ``POST /consumers/{group}`` 
+                                      with ``Content-Type: application/vnd.kafka.v2+json`` header
+  ``consumer.delete``                 ``DELETE /consumers/{group}/instances/{instance}``
```

Comment:
I think this should be `consumer.delete+v2`. Same `+v2` needed on the next few as well.